src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.Linux.cs

```diff
@@ -251,6 +249,7 @@ private sealed class RunningInstance
                 _includeSubdirectories = includeSubdirectories;
                 _notifyFilters = notifyFilters;
                 _cancellationToken = cancellationToken;
+                FileSystemWatcher.CaseSensitive = true;
```

Comment:
Ok.  I don't feel strongly about it.  We can always change it in the future if we so desire.